### PR TITLE
Class that creates a presetQueue 

### PR DESCRIPTION
**Pull request by NaomidR**
We now have a presetQueue that can store presets in order.

### CODE REVIEW

**Comment by bvanwijk** [comment]
Looks good, if someone else agrees I think this can be merged.

File: backend/src/main/java/com/benine/backend/performance/PresetQueue.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+import java.util.ArrayList;
+
+
+/**
+ * Class that handles making a queue of presets.
+ * Created by Team BeNine.
+ */
+public class PresetQueue {
+
+  private ArrayList<Preset> queue = new ArrayList<Preset>();
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/performance/PresetQueue.java]: I think you can remove `Preset` from the diamonds in `new ArrayList<Preset>`

**Comment by jochem725** [comment]
Looks good, I've placed a small comment after this has been fixed this can be merged.

**Comment by egedebruin** [comment]
Good work, I will merge this.

**Pull request merged**